modules/cloudaccess/examples/complete/outputs.tf

```diff
@@ -1,6 +1,6 @@
 output "managed_identity_client_id" {
   value = module.wayfinder_cloudaccess.managed_identity_client_id
 }
-output "managed_identity_principal_id" {
-  value = module.wayfinder_cloudaccess.managed_identity_principal_id
+output "managed_identity_tenant_id" {
```

Comment:
Need to autogenerate / update the readme file for this.

Reply:
There is no README for the example.......... 👀 😞 what do I need to run?

Reply:
Try this: `terraform-docs markdown table --output-file ${PWD}/README.md --output-mode inject .`

Reply:
The CI workflow isn't validating this examples dir so ignore